package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@nib-styles/alerts",
-  "version": "2.0.4",
+  "version": "3.0.0",
```

Comment:
Does this need to be a major bump? As far as I can tell this only uses the `mixins` from the `v2-icons` repo so it shouldn't break anything...